src/config/api/polkadot/connectApi.ts

```diff
@@ -52,29 +52,72 @@ const loadAccounts = async (api: ApiPromise) => {
   );
 };
 
-const fallBackConnection = async (networkIdx: number): Promise<void> => {
-  await Promise.race([
-    providerEndpoints[networkIdx].endpoints.map(async (it) => {
-      try {
+const fallbackConnection = async ({
```

Comment:
please add a comment to the function's behavior (and other functions too)

Reply:
Fixed.

https://github.com/AstarNetwork/astar-apps/pull/375/commits/3390ac6fcab9fb9a0e54172f2a1b9753a0cf0969
https://github.com/AstarNetwork/astar-apps/pull/375/commits/0560a87967d0baafb0f6cea7d94bbdf5033276dd